Mastodon/Protocol/Provider/DataSourceProvider+NotificationTableViewCellDelegate.swift

```diff
@@ -494,21 +480,18 @@ extension NotificationTableViewCellDelegate where Self: DataSourceProvider & Aut
                 return
             }
             switch item {
-            case .status(let status):
-                await DataSourceFacade.coordinateToStatusThreadScene(
-                    provider: self,
-                    target: .status,    // remove reblog wrapper
-                    status: status
-                )
-            case .user(let user):
-                await DataSourceFacade.coordinateToProfileScene(
-                    provider: self,
-                    user: user
-                )
-            case .notification:
-                assertionFailure("TODO")
-            default:
-                assertionFailure("TODO")
+                case .status(let status):
```

Comment:
I'm just wondering about the indentations here because for me, and I think with the Xcode default settings, the above is the correctly indented version.